src/Promitor.Integrations.Sinks.Statsd/Configuration/StatsdSinkConfiguration.cs

```diff
@@ -5,5 +5,8 @@ public class StatsdSinkConfiguration
         public string Host { get; set; }
         public int Port { get; set; } = 8125;
         public string MetricPrefix { get; set; }
+        public string FormatterType { get; set; } = StatsdFormatterTypes.DEFAULT;
```

Comment:
Can you add unit tests for parsing the configuration please? This should already be there so you can just extend it